pom.xml

```diff
@@ -71,8 +77,18 @@
             <version>1.5.1</version>
             <optional>true</optional>
         </dependency>
+        <dependency>
```

Comment:
Surprised that you needed a new JUnit. Why is this needed? We should always use the version in the parent pom

The version the parent POM lists is:
```
     <dependency>
        <groupId>junit</groupId>
        <artifactId>junit</artifactId>
        <version>4.12</version>
      </dependency>
```